test/tests/classesTest.js

```diff
@@ -96,12 +75,13 @@ module.exports = function (request, db) {
         });
 
         it('should GET my classes', function (done) {
-            request.get('/api/games/' + idGame + '/versions/' + idVersion + '/classes/my')
+            request.get('/api/classes/my')
                 .expect(200)
                 .set('X-Gleaner-User', 'Teacher1')
                 .end(function (err, res) {
                     should.not.exist(err);
                     should(res).be.Object();
+                    console.log(res.body);
                     should.equal(res.body.length, 1);
```

Comment:
Fixed in: https://github.com/e-ucm/rage-analytics-backend/pull/105/commits/57016fb261927968c5d2591adf289663f3b0be4b